pom.xml

```diff
@@ -442,7 +442,7 @@
       <id>default-modules</id>
       <activation>
         <property>
-          <name>!default-modules</name>
+          <name>!default-modules</name> <!-- Defaults to ON, when set it will turn off -->
```

Comment:
Is er een usecase te verzinnen waarin je deze modules (inclusief core dus) uit zou willen zetten?
Zo nee, dan lijkt het me beter om ook niet de last te hebben van deze switch
Zo ja, dan zou ik de property zoiets noemen als 'suppress-default-modules' of 'no-default-modules'